src/main/java/seedu/address/logic/parser/FindCommandParser.java

```diff
@@ -20,15 +20,38 @@
      * @throws ParseException if the user input does not conform the expected format
      */
     public FindCommand parse(String args) throws ParseException {
-        String trimmedArgs = args.trim();
-        if (trimmedArgs.isEmpty()) {
+        String trimmedArgs = args.trim().replace(FindCommand.COMMAND_WORD, "");
```

Comment:
Why not use the argument multimap? I believe it makes the code follow OOP more, and reduces the complexity of the parse method.

Reply:
Hmm is this a typo here, maybe you meant improve?